### ISSUE TITLE
Please support UIA_HeadingLevelPropertyId

### ISSUE THREAD

**Issue by jschroedl** [Emil-18/enhanced-object-navigation#3]
This is a nice NVDA extension. Trying it with our application, we can navigate among tables with 't' but cannot navigate among headings with 'h' because `createHeadingWalker()` is looking for a `UIA_AriaRolePropertyId` of "heading". That is fine for web pages but ours is a desktop app written using the Windows Presentation Framework (WPF) framework which does not expose Aria properties to desktop software. Instead, we set the heading level on elements which turn into `UIA_HeadingLevelPropertyId`. Could you update the add-on to also navigate among those headings?

**Comment by Emil-18** [comment]
@jschroedl I will see what I can do. In the meantime, you could try turning off "use UIA when navigating" in the add-on settings, and see if it works then. Could you send me the application, so I can take a look at it?

**Comment by jschroedl** [comment]
Thanks @Emil-18 - I can't give a copy of my specific app b/c it's under active development but I could probably make a WPF application which shows the same issue as a test case.

**Comment by jschroedl** [comment]
Hi @Emil-18, I created a repo with a test app. I also included a built binary in case you don't have Visual Studio or the free Visual Studio Community Edition. It's a simple WPF app with one main heading (level 1) and a number of level 2 headings. You can tab among them just fine. But if I use your add-in and enter Navigation mode and press 'h', the headings are not visited. As the README.md mentions, Narrator handles them fine in its scan mode.

[https://github.com/jschroedl/HeadingLevelTest](https://github.com/jschroedl/HeadingLevelTest)

**Comment by jschroedl** [comment]
> could try turning off "use UIA when navigating" in the add-on settings

Forgot to mention that this setting didn't make a difference for me. Our app is fully using UIA since WPF uses that for its AutomationPeer implementation.

**Comment by jschroedl** [comment]
Just a note that this fix allows your extension to work well with our desktop app. Thanks again for the fast update!
